build.js

```diff
@@ -0,0 +1,125 @@
+import * as rolldown from 'rolldown';
+
+const externalDeps = [
```

Comment:
why not automate this by reading package.json devdeps vs deps? (and then keep the node stuff, I suppose

e.g.:


external: (x) => {
  return dependencies.includes(x) 
         || x.startsWith('node:') 
         || isKownNodeBuiltIn(x);
}

Reply:
not all deps bundlable(, I'm using array to be able simply turn on/off dependency from bundling (simplify debugging)